src/main.rs

```diff
@@ -56,34 +59,73 @@ fn main() {
                 // Can this ever happen?
                 Ok(())
             } else {
-                println!("Fuzzing found errors!");
+                if let Some(ref mut terminal) = term_stdout {
+                    let _ = terminal.fg(term::color::YELLOW);
+                    println!("Fuzzing found errors!");
+                    let _ = terminal.reset();
+                } else {
+                    println!("Fuzzing found errors!");
+                }
```

Comment:
Can we move all this logic into a separate function now that it's getting a little noise? Something like:

```rust
fn print_found_errors() {
  let mut term_stdout = term::stdout();
  let mut term_stderr = term::stderr();
  if let Some(ref mut terminal) = term_stdout {
      let _ = terminal.fg(term::color::YELLOW);
      println!("Fuzzing found errors!");
      let _ = terminal.reset();
    } else {
      println!("Fuzzing found errors!");
    }
  }
}
```